.github/workflows/qa.yml

```diff
@@ -0,0 +1,80 @@
+name: QA
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  schedule:
+    - cron: '40 15 * * *'
```

Comment:
Why 40?

Reply:
I've set it to a time that it used to run on travis (from travis logs).

Reply:
Ah ok :-) We don't control the scheduled time at Travis, so I suggest to keep things simple and use noon or midnight.

Reply:
done